### PR TITLE
11 create a plain omap docker image on dockerhub

### LINKED ISSUE #11: Create a plain OMAP Docker Image on DockerHub
Pretty self explaining, I guess.

### PR DESCRIPTION
**Pull request by MarcoNastasi**
Made adjustments for docker and made an initial push to pragmaticindustriesgmbh/open-machine-platform

Post-Push I'll add a docker-release action I think.

### CODE REVIEW

**Comment by JulianFeinauer** [comment]
@MarcoNastasi we could also use DockerHubs build feature to build it there... but.. I dont care that much tbh : )

File: omap/modules/base_settings.py [hunk 1]
```diff
@@ -68,7 +69,8 @@
         },
     },
 ]
-
+WSGI_APPLICATION = "omap.modules.wsgi.application"
+ASGI_APPLICATION = "sockpuppet.routing.application"
```
Comment (JulianFeinauer) [inline omap/modules/base_settings.py]: shouldnt we have our own asgi application here?
we dont use sockpuppet in OMAP, or?

**Comment by MarcoNastasi** [comment]
@JulianFeinauer Fair point. Do we want it to be built for every push to develop or are we going to have releases? (Or both?)

File: omap/modules/base_settings.py [hunk 2]
```diff
@@ -52,6 +52,7 @@
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
 ]
+ROOT_URLCONF = "omap.modules.module_urls"
```
Comment (JulianFeinauer) [inline omap/modules/base_settings.py:55]: hm... why is that in here? or rather... why is this a change here? Wasnt that in here already?

**Review by JulianFeinauer — COMMENTED** [review]
Looks good... just minor things :)

File: omap/modules/base_settings.py (hunk 2 above)
Comment (MarcoNastasi) [reply omap/modules/base_settings.py:55]: @JulianFeinauer It was indeed not!

File: omap/modules/base_settings.py (hunk 1 above)
Comment (MarcoNastasi) [reply omap/modules/base_settings.py]: @JulianFeinauer Thats a fair point, I just copied those over, I'll fix that.

**Pull request merged**